src/i18next/init-i18next.js

```diff
@@ -0,0 +1,55 @@
+import i18n from "i18next";
+import i18nextXhrBackend from "i18next-xhr-backend";
+import { initReactI18next } from "react-i18next";
+import ICU from "i18next-icu";
+
+(i18n.default || i18n)
+  .use(i18nextXhrBackend)
+  .use(initReactI18next)
+  .use(ICU)
```

Comment:
Transifex only supports plural format via the ICU standard for plurals. We are using i18next-icu so that i18next can understand this format.